docs/how-to-guides/develop-standard-flow.md

```diff
@@ -0,0 +1,312 @@
+The classification will be based on the url, the webpage text content summary, or both.
+
+Here are a few examples:
+{% for ex in examples %}
+URL: {{ex.url}}
+Text content: {{ex.text_content}}
+OUTPUT:
+{"category": "{{ex.category}}", "evidence": "{{ex.evidence}}"}
+
+{% endfor %}
+
+For a given URL : {{url}}, and text content: {{text_content}}.
+Classify above url to complete the category and indicate evidence.
+OUTPUT:
+```
+
+### Define Python node interface
+Python node might have multiple inputs and outputs. Define inputs and outputs as shown below. 
+If you have multiple outputs, remember to make it a dictionary so that the downstream node can call each key separately.
+For example:
+
+```python
+import json
+from promptflow import tool
+
+@tool
+def convert_to_dict(input_str: str, input_str2: str) -> dict:
+    try:
+        print(input_str2)
+        return json.loads(input_str)
+    except Exception as e:
+        print("input is not valid, error: {}".format(e))
+        return {"category": "None", "evidence": "None"}
+```
+
+### Link nodes together
+After the interface is defined, you can use:
+
+- ${inputs.key} to link with flow input.
+- ${upstream_node_name.output} to link with single-output upstream node.
+- ${upstream_node_name.output.key} to link with multi-output upstream node.
+
+Below are common scenarios for linking nodes together.
+
+### Scenario 1 - Link LLM node with flow input and single-output upstream node
+After you add a new LLM node and edit the prompt file like [Define LLM node interface](#define-llm-node-interface), 
+three inputs called `url`, `examples` and `text_content` are created in inputs section.
+
+::::{tab-set}
+:::{tab-item} CLI
+:sync: CLI
+You can link the LLM node input with flow input by `${inputs.url}`. 
+And you can link `examples` to the upstream `prepare_examples` node and `text_content` to the `summarize_text_content` node 
+by `${prepare_examples.output}` and `${summarize_text_content.output}`. 
+```yaml
+- name: classify_with_llm
+  type: llm
+  source:
+    type: code
+    path: classify_with_llm.jinja2
+  inputs:
+    deployment_name: text-davinci-003
+    suffix: ""
+    max_tokens: 128
+    temperature: 0.2
+    top_p: 1
+    echo: false
+    presence_penalty: 0
+    frequency_penalty: 0
+    best_of: 1
+    url: ${inputs.url}    # Link with flow input
+    examples: ${prepare_examples.output} # Link LLM node with single-output upstream node
+    text_content: ${summarize_text_content.output} # Link LLM node with single-output upstream node
+```
+:::
+
+:::{tab-item} VS Code Extension
+:sync: VS Code Extension
+In the value drop-down, select `${inputs.url}`, `${prepare_examples.output}` and `${summarize_text_content.output}`, then 
+you'll see in the graph view that the newly created LLM node is linked to the flow input, upstream `prepare_examples` and `summarize_text_content` node. 
+
+![scenario1](../media/how-to-guides/develop-standard-flow/scenario1.png)
```

Comment:
Lets give the scenario a name, avoid using number in file name

Reply:
updated